lisa/features/security_profile.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from dataclasses import dataclass
+from functools import partial
+from typing import Any, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.feature import Feature
+
+FEATURE_NAME_SECURITY_PROFILE = "Security_Profile"
+
+
+@dataclass_json()
+@dataclass()
+class SecurityProfileSettings(schema.FeatureSettings):
+    type: str = FEATURE_NAME_SECURITY_PROFILE
+    secure_boot_enabled: bool = False
+    vtpm_enabled: bool = False
```

Comment:
The two variables are set to true together always, so it doesn't need to split to two variables.

Reply:
I am not 100% sure, since from the portal, we can just enable one of them.

Reply:
We can put them together firstly. If it's needed, then split it. We're not create the TVM feature, it's for integration testing.